<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KcmPacmanRepoEditor</class>
 <widget class="QWidget" name="KcmPacmanRepoEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>687</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QTableView" name="tableView">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>66</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>10</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="edit">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Edit</string>
       </property>
       <property name="icon">
        <iconset theme="edit-rename">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset theme="list-add">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remove">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="icon">
        <iconset theme="list-remove">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="moveUp">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Move Up</string>
       </property>
       <property name="icon">
        <iconset theme="go-up">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="moveDown">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Move Down</string>
       </property>
       <property name="icon">
        <iconset theme="go-down">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Available Repositories</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="loadBackup">
       <property name="text">
        <string>Load a backup file</string>
       </property>
       <property name="icon">
        <iconset theme="document-open">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
